src/openapi/common-responses.ts

```diff
@@ -89,11 +89,31 @@ export const internalServerErrorResponse: OpenAPIV3.ResponseObject = {
     },
 };
 
+export const notImplementedError: OpenAPIV3.ResponseObject = {
+    description:
+        'The functionality you are requesting is not implemented by the server. This could be due to a misconfiguration when starting the server.',
+    content: {
+        'application/json': {
+            schema: {
+                type: 'object',
+                required: ['error'],
+                properties: {
+                    error: { type: 'string' },
```

Comment:
Could we describe what the `error` property does here? Something like:

```suggestion
                    error: { type: 'string', description: 'A description of the error that occurred.' },
```